economy/models.py

```diff
@@ -152,31 +152,46 @@ class ProductOrder(models.Model):
     )
 
     order_size = models.IntegerField(default=1)
-    amount = models.IntegerField()
 
-    purchase = models.ForeignKey(
```

Comment:
What is the new relation between `product orders` and the rest of the models? Could you please update the documentation in `api/api_docs.py`?

Reply:
The new product order would be an aggregation of the old product_order and the Purchase model. I would start doing a more thorough process of deprecating things when or if this PR goes through.